packages/mobile/src/common/enums/http/http-error.ts

```diff
@@ -0,0 +1,13 @@
+enum HttpError {
```

Comment:
`packages/shared/src/common/enums/http/http-status-code.ts`

Reply:
This is no longer relevant due to a code refactor